gravitee-am-ui/src/app/domain/settings/audits/audits.component.html

```diff
@@ -169,12 +169,21 @@ <h1>Audit log</h1>
       </ngx-datatable-column>
       <ngx-datatable-column [sortable]="true" name="Target" [flexGrow]="2" prop="target.sortDisplayName">
         <ng-template let-row="row" ngx-datatable-cell-template>
-          <a *ngIf="row.target" [routerLink]="getTargetUrl(row)" [queryParams]="getTargetParams(row)"
+          <a
+            *ngIf="row.target && !isDomainAuditOnOrganizationLevel(row)"
+            [routerLink]="getTargetUrl(row)"
+            [queryParams]="getTargetParams(row)"
             >{{ row.target?.displayName
             }}<span *ngIf="row.target.alternativeId">
               | <small>{{ row.target?.alternativeId }}</small></span
             ></a
           >
+          <span *ngIf="row.target && isDomainAuditOnOrganizationLevel(row)">
+            {{ row.target?.displayName
+            }}<span *ngIf="row.target.alternativeId">
+              | <small>{{ row.target?.alternativeId }}</small></span
+            >
+          </span>
```

Comment:
suggestion: put intermediate div element with ngIf="row.target". Then in "a" and "span" the conditions simplify.